service/src/main/java/bio/terra/workspace/common/utils/AwsUtils.java

```diff
@@ -597,15 +597,15 @@ private static void deleteS3Objects(
     }
   }
 
-  // AWS Sagemaker Notebook
+  // AWS SageMaker Notebook
 
   // TODO(TERRA-500) Move notebook functions below to CRL
 
   /**
-   * Create a AWS sagemaker notebook
+   * Create a AWS sageMaker notebook
```

Comment:
As a proper noun In comments/error strings/text I think this should be `SageMaker`, not `sageMaker`.

IMO `sageMaker___` is fine as a var name though

Reply:
Agree, this was missed due to a project wide find and replace